### PR TITLE
fix(windows): windows image changed

### PR DESCRIPTION
**Pull request by lukidzi**
Previous image seems to be removed, changed to image for other one with container support

### CODE REVIEW

**Review by slonka — APPROVED** [review]
Why do we need `/image_id` at the end now?

File: terraform/windows.tf [hunk 1]
```diff
@@ -1,3 +1,3 @@
 data "aws_ssm_parameter" "windows" {
-  name = "/aws/service/ami-windows-latest/Windows_Server-2019-English-Core-ContainersLatest"
+  name = "/aws/service/ami-windows-latest/Windows_Server-2019-English-Core-EKS_Optimized-1.23/image_id"
```
Comment (michaelbeaumont) [inline terraform/windows.tf:2]: just so I understand, ContainersLatest doesn't work even with `/image_id`? Surprising behavior in terms of backwards compatibility

Comment (slonka) [reply terraform/windows.tf:2]: > ContainersLatest doesn't work even with /image_id

I think it no longer exists.

**Comment by bartsmykla** [comment]
Superseded by: https://github.com/kumahq/envoy-builds/pull/40

**Pull request closed without merging**